casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -155,14 +155,14 @@ class RuntimeManager private (val emptyStateHash: ByteString, runtimeContainer:
             val cost       = CostAccount.toProto(costAlg.getCost().unsafeRunSync)
             val deployCost = DeployCost().withDeploy(deploy).withCost(cost)
             if (errors.isEmpty)
-              eval(rest, reducer, errorLog, costAlg, deployCost +: accCost)
+              eval(rest, reducer, errorLog, costAlg, accCost :+ deployCost)
             else
-              Left((deploy, errors, accCost))
+              Left((deploy, errors, accCost :+ deployCost))
           case Failure(ex) =>
             val otherErrors = errorLog.readAndClearErrorVector()
             val cost        = CostAccount.toProto(costAlg.getCost().unsafeRunSync)
             val deployCost  = DeployCost().withDeploy(deploy).withCost(cost)
-            Left((deploy, ex +: otherErrors, deployCost +: accCost))
+            Left((deploy, ex +: otherErrors, accCost :+ deployCost))
```

Comment:
Shouldn't the errors be reversed as well?

Reply:
Seems less critical because they all apply to the same deploy. But sure, we can append there too.